modules/button/css/lib/accessibility.scss

```diff
@@ -33,7 +34,7 @@
     }
   }
 
-  .wdc-btn-delete:not([disabled]):not(.wdc-btn-disabled):not([tabindex='-1']):focus {
+  .wdc-btn-delete:not([disabled]):not(.wdc-btn-disabled):not([tabindex="-1"]):focus {
```

Comment:
this is prettier... sorry

Reply:
Are you sure? If I check out this file from master and run it through it, this doesn't happen

Reply:
May be related to https://github.com/Workday/canvas-kit/pull/348